src/_utils/constants/index.js

```diff
@@ -38,23 +40,76 @@ const TYPE = {
 }
 
 const UNIT = {
-  HEIGHT: {
+  DISTANCE: {
     CM: 'cm',
     IN: 'in',
```

Comment:
Shouldn't we split height and distance? cm and in will probably only be used for height while the others for distance.

Reply:
As discussed probably better to use `length`